### PR TITLE
watermarking for thumbnails

### PR DESCRIPTION
**Pull request by marsha97**
#105

### CODE REVIEW

File: thumbnails/processors.py [hunk 1]
```diff
@@ -45,6 +45,26 @@ def set_quality(image, **kwargs):
     return image
 
 
+def add_watermark(image, **kwargs):
```
Comment (selwin) [inline thumbnails/processors.py:48]: Bisa tambah docs @marsha97 ?

Comment (marsha97) [reply thumbnails/processors.py:48]: ok pak 👌

File: README.md [hunk 2]
```diff
@@ -64,8 +64,10 @@ THUMBNAILS = {
     thumbnails.processors.rotate(degrees)
     thumbnails.processors.flip(direction)
     thumbnails.processors.crop(width, height, center)
+    thumbnails.processors.add_watermark(path, width, height)
```
Comment (selwin) [inline README.md:67]: Mestinya cuma terima `image_path`

File: thumbnails/processors.py [hunk 3]
```diff
@@ -45,6 +45,26 @@ def set_quality(image, **kwargs):
...
+    watermark_path = kwargs["path"]
```
Comment (selwin) [inline thumbnails/processors.py:49]: `path` -> `image_path`

File: README.md (hunk 2 above)
Comment (selwin) [reply README.md:67]: Tambahin contoh penggunaan di `settings.py`.

File: README.md [hunk 4]
```diff
@@ -52,6 +52,12 @@ THUMBNAILS = {
                 {'PATH': 'thumbnails.processors.resize', 'width': 20, 'height': 20},
                 {'PATH': 'thumbnails.processors.flip', 'direction': 'horizontal'}
             ],
+        },
+        'watermarked': {
+            'PROCESSORS': [
+                {'PATH': 'thumbnails.processors.resize', 'width': 20, 'height': 20},
+                {'PATH': 'thumbnails.processors.add_watermark', 'image_path': 'watermark.png'}
```
Comment (selwin) [inline README.md:59]: Tambahin comment kalo `watermark.png should be of the same size with resulting thumbnail (20x20)`

File: README.md (hunk 2 above)
Comment (selwin) [reply README.md:73]: Perlu hapus arguments `width` dan `height`.

File: thumbnails/processors.py [hunk 5]
```diff
@@ -45,6 +45,25 @@ def set_quality(image, **kwargs):
     return image
 
 
+def add_watermark(image, **kwargs):
```
Comment (selwin) [inline thumbnails/processors.py:48]: Rename `image` ke `image_path`

Comment (marsha97) [reply thumbnails/processors.py:48]: Sorry pak @selwin , image ini udah object imagenya pak, jadi nggak saya rename dulu

**Pull request merged**